src/Compatibility/Core/src/Android/Renderers/WebViewRenderer.cs

```diff
@@ -48,7 +48,8 @@ void LoadUrl(string url, bool fireNavigatingCanceled)
 			if (!fireNavigatingCanceled || !SendNavigatingCanceled(url))
 			{
 				_eventState = WebNavigationEvent.NewPage;
-				if (url != null && !url.StartsWith('/') && !Uri.IsWellFormedUriString(url, UriKind.Absolute))
+				string? spacesEscapedUrl = url?.Replace(" ","%20", StringComparison.InvariantCulture);
```

Comment:
Is there no Uri.EncodeSomething that is safer? What about other "special" characters in the uri?

Reply:
That is a bit problematic: https://github.com/dotnet/maui/pull/24002#issuecomment-2267527015

Reply:
Can we only encode whatever comes after the base URL? So only the query string basically?

Reply:
Imo, great idea!